apps/mudita-center-e2e/src/specs/news/news-check-offline.e2e.ts

```diff
@@ -0,0 +1,106 @@
+import exp from "constants"
+import screenshotHelper from "../../helpers/screenshot.helper"
```

Comment:
unused too

Reply:
it was used in online mode test so I just copied it